messages/src/lib.rs

```diff
@@ -52,6 +55,7 @@ pub enum AriesMessage {
     Connection(Connection),
     Revocation(Revocation),
     CredentialIssuance(CredentialIssuance),
+    CredentialIssuanceV2(CredentialIssuanceV2),
```

Comment:
I also realized now that this is not the intended way to deal with this. The `CredentialIssuance` variant should contain the underlying versions in a separate enum.

Reply:
kk i've gone ahead with this change. A note tho; it made the `transit_to_aries_msg` macros not directly work. i.e. 
```
transit_to_aries_msg!(
    OfferCredentialV1Content: OfferCredentialV1Decorators,
    CredentialIssuanceV1
);
```

i had to add the following explicit `From` statements to complete the `From` chain:
```
impl From<CredentialIssuanceV1> for AriesMessage {
    fn from(value: CredentialIssuanceV1) -> Self {
        AriesMessage::CredentialIssuance(CredentialIssuance::V1(value))
    }
}

impl From<CredentialIssuanceV2> for AriesMessage {
    fn from(value: CredentialIssuanceV2) -> Self {
        AriesMessage::CredentialIssuance(CredentialIssuance::V2(value))
    }
}
```